superset/config.py

```diff
@@ -323,7 +323,7 @@ def _try_json_readsha(  # pylint: disable=unused-argument
     # When True, this escapes HTML (rather than rendering it) in Markdown components
     "ESCAPE_MARKDOWN_HTML": False,
     "SIP_34_ANNOTATIONS_UI": False,
-    "DASHBOARD_NATIVE_FILTERS": False,
+    "DASHBOARD_NATIVE_FILTERS": True,
```

Comment:
Please use local configuration to set the feature flag, we want this `False` by default until later stages of the project.

If using docker, you can follow [these docs](https://github.com/apache/incubator-superset/blob/master/docker/README.md) to set local config.

That said, I'm not planning on letting things like this block merge, since it's a PR into a WIP branch 😁

Reply:
fixed